src/hevm/src/EVM/TTY.hs

```diff
@@ -945,27 +946,36 @@ drawTracePane s =
           1
 
   in case view uiShowMemory s of
-    True ->
-      hBorderWithLabel (txt "Calldata")
-      <=> str (prettyIfConcrete (view (state . calldata) vm))
-      <=> hBorderWithLabel (txt "Returndata")
-      <=> str (prettyIfConcrete (view (state . returndata) vm))
-      <=> hBorderWithLabel (txt "Output")
-      <=> str (maybe "" show (view result vm))
-      <=> hBorderWithLabel (txt "Cache")
-      <=> str (show (view (cache . path) vm))
-      <=> hBorderWithLabel (txt "Path Conditions")
-      <=> (str $ show $ view constraints vm)
-      <=> hBorderWithLabel (txt "Memory")
-      <=> viewport TracePane Vertical
-            (str (prettyIfConcrete (view (state . memory) vm)))
+    True -> viewport TracePane Vertical $
+        hBorderWithLabel (txt "Calldata")
+        <=> ourWrap (prettyIfConcrete (view (state . calldata) vm))
+        <=> hBorderWithLabel (txt "Returndata")
+        <=> ourWrap (prettyIfConcrete (view (state . returndata) vm))
+        <=> hBorderWithLabel (txt "Output")
+        <=> ourWrap (maybe "" show (view result vm))
+        <=> hBorderWithLabel (txt "Cache")
+        <=> ourWrap (show (view (cache . path) vm))
+        <=> hBorderWithLabel (txt "Path Conditions")
+        <=> (ourWrap $ show $ view constraints vm)
+        <=> hBorderWithLabel (txt "Memory")
+        <=> (ourWrap (prettyIfConcrete (view (state . memory) vm)))
```

Comment:
On this note, we should merge Zoe's pending changes and auto-indent everything :) Seems like there is a mix of indentation going on sometimes.